### DISCUSSION TITLE
changing Opacity of image

### DISCUSSION THREAD

**Discussion by Flinner** [emacs-dashboard/dashboard#376] [Q&A]
Now that emacs has opacity,
```elisp
  (set-frame-parameter nil 'alpha-background 0.9)
```

can the image have matching opacity as well? 
![image](https://user-images.githubusercontent.com/85732279/165759107-0f9743a6-c4b6-4832-9f4b-fbfd19198535.png)

**Comment by jcs090218** [comment]
Thanks for informing us this feature!

I will look into it once I have time! :)
